googletest.sh

```
@@ -3,7 +3,7 @@ version: "1.17.0"
 source: https://github.com/google/googletest
 tag: v1.17.0
 build_requires:
- - "GCC-Toolchain:(?!osx)"
+ - GCC-Toolchain
```

Shellcheck (0.11.0)

[warning] This flag is used as a command name. Bad line break or missing [ .. ]?

(SC2215)